### PR TITLE
Feat(Openebs-pool-disk-loss): Adding event support for openebs pool disk loss experiment

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: Raj <raj.das@mayadata.io>

**What this PR does / why we need it**:
- Adding event support for openebs pool disk loss experiment

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml [hunk 1]
```diff
@@ -57,6 +57,15 @@
       - name: AWS authentication
         include_tasks: "/utils/cloud/aws/aws_configure.yml"
         when: "cloud_platform == 'AWS'"
+        
+      ## RECORD EVENT FOR PRE-CHAOS CHECK
+      - include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
```
Comment (ispeakc0de) [inline experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml]: Can you please add this task just after checking the running status of AUT and auxiliary apps?

File: experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml [hunk 2]
```diff
@@ -126,7 +135,17 @@
...
-      # POST CHAOS CHECKS
+      # POST CHAOS 
+      
+      ## RECORD EVENT FOR POST-CHAOS CHECK
+      - include_tasks: /utils/common/generate-kubernetes-chaos-events.yml
+        vars:
+          stage: "PostChaosCheck"
+          exp_pod_name: "{{ chaos_pod_name }}"
+          engine_ns: "{{ a_ns }}"
+          message: "AUT is Running successfully"
```
Comment (ispeakc0de) [inline experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml]: Can you please add this task just after checking the running status of AUT and auxiliary apps?

File: experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml (hunk 1 above)
Comment (ispeakc0de) [reply experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml]: @rajdas98 Please don't resolve the conversation without solving it.

Comment (rajdas98) [reply experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml]: My bad forget to push. Made the changes

Comment (rajdas98) [reply experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml]: Can you check now?

**Comment by rajdas98** [comment]
My bad forgot to push. Made the changes

**Comment by ispeakc0de** [comment]
@rajdas98  Can you please attach the logs for reference?

**Comment by ksatchit** [comment]
Can we add a description ?

**Pull request merged**